src/fides/api/main.py

```diff
@@ -190,18 +189,16 @@ def configure_routes() -> None:
 @app.on_event("startup")
 async def setup_server() -> None:
     "Run all of the required setup steps for the webserver."
-    logger.warning(
-        "Startup configuration: reloading = %s, dev_mode = %s",
-        CONFIG.hot_reloading,
-        CONFIG.dev_mode,
+    log.warning(
+        f"Startup configuration: reloading = {CONFIG.hot_reloading}, dev_mode = {CONFIG.dev_mode}",
```

Comment:
A note to say that `loguru` appears not to support string substitution, so I've updated them to `f"..."` strings.

Reply:
In ops, we'd moved to `%-style` formatting to better support masking PII https://github.com/ethyca/fidesops/pull/1132/. I think there's a lot of good reasons to prefer %-style formatting instead.

Reply:
@pattisdr is correct here, for logging specifically `%s` is actually the most optimal way due to inner workings of Python

https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs

Reply:
@seanpreston I don't think it's been logged anywhere...

Reply:
I totally agree here, we ought to be rallying behind that as an eng. team.